istqb-ct-mbt-complete-guide.md

```
@@ -0,0 +1,1609 @@
+**構成要素:**
+- **状態（State）:** システムが取りうる各状態（例：ログイン前、ログイン中、エラー）
+- **遷移（Transition）:** ある状態から別の状態への移動
+- **イベント（Event）:** 遷移を引き起こすトリガー（例：ボタンクリック）
+- **アクション（Action）:** 遷移時に実行される処理
+- **ガード（Guard）:** 遷移条件（例：[パスワードが正しい場合]）
+
+**具体例：ログイン機能のFSM**
+
+> 状態: S0=初期画面, S1=ログイン成功, S2=エラー表示, S3=アカウントロック
+
+>          ログイン試行[PW正]
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)